app/components/main-hero/template.hbs

```diff
@@ -26,6 +26,4 @@
       User Guide
     {{/link-to}}
   </div>
-
-  {{main-hero/meetup}}
```

Comment:
I did think about that but then thought we could always add new css pertaining to how we want to showcase that meetup (since next time we might want to show it as a banner or have different visuals for it) 
Personally, I think we should remove the (now) dead code 

It's not too big a change, but that works in both side about whether we should keep it or not haha

Reply:
If it's something we use often, sure 
I can even refactor the template to be a more generic one

Reply:
I was thinking that a better approach to this might be to make the template a generic one and keep the files

Reply:
I agree with @suchitadoshi1987 on this one, let's change this PR to just remove the meetup component **call**

In the learning team stuff we have more generic components for announcements that we can configure from data (somewhere) and we can follow up with a refactor later 👍  maybe we can make an issue and mark it as "help wanted" 😄

Reply:
Ah, thanks for pitching in
I'll do that instead then :)

Reply:
Done